messaging/client.go

```diff
@@ -335,12 +339,12 @@ func (c *Client) do(method, path string, values url.Values, contentType string,
 }
 
 // Conn returns a connection to the broker for a given topic.
-func (c *Client) Conn(topicID uint64) *Conn {
+func (c *Client) Conn(topicID uint64, dataURL *url.URL) *Conn {
```

Comment:
I know it's a method, not a free function, but I still think the name could be different.